### DISCUSSION TITLE
阿里云盘现在运行申请最长下载时间到4小时

### DISCUSSION THREAD

**Discussion by honokachannel** [AlistGo/alist#5669] [General]
![$GQ2V_U~_2 KE)F}2})37Z6](https://github.com/alist-org/alist/assets/152259293/c1836a3f-2270-4cb4-aaef-73c84aa592b5)

**Comment by LGuangming** [comment]
对Alist和自己申请app对接的影响是啥，我要正在看一个电影 连接还能失效？不太清楚这个下载链接失效什么意思

**Comment by ypq123456789** [comment]
怎么申请？我有自己的开发者账号
